server/src/main/java/org/opentosca/toscana/plugins/cloudformation/visitor/CheckModelRelationshipVisitor.java

```diff
@@ -28,7 +29,8 @@ public CheckModelRelationshipVisitor(TransformationContext context) {
     public void visit(ConnectsTo relation) {
         RootNode source = topology.getEdgeSource(relation);
         RootNode target = topology.getEdgeTarget(relation);
-        if (!(source instanceof WebApplication && target instanceof MysqlDatabase)) {
+        if (!(source instanceof WebApplication && target instanceof MysqlDatabase) &&
```

Comment:
If the second condition is true, the first condition is also true. You might want to omit the first statement.